src/App.test.js

```diff
@@ -31,8 +31,9 @@ let loginReducer = (state = initialState, action) => {
   }
 }
 
+// , profileReducer
 
-const store = createStore(combineReducers({loginReducer,plaidReducer,registerReducer,profileReducer}), applyMiddleware(thunk, logger));
+const store = createStore(combineReducers({loginReducer,plaidReducer,registerReducer}), applyMiddleware(thunk, logger));
```

Comment:
Formatting is a little weird here. Can we run prettier on all the files before committing?